editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -610,23 +610,29 @@ void CanvasItemEditor::_find_canvas_items_at_pos(const Point2 &p_pos, Node *p_no
 	if (!p_node) {
 		return;
 	}
-	if (Object::cast_to<Viewport>(p_node)) {
-		return;
-	}
 
 	const real_t grab_distance = EDITOR_GET("editors/polygon_editor/point_grab_radius");
 	CanvasItem *ci = Object::cast_to<CanvasItem>(p_node);
 
+	Transform2D p_xform = p_canvas_xform;
```

Comment:
```suggestion
	Transform2D xform = p_canvas_xform;
```
`p_` prefix is reserved for parameters.

Reply:
updated